components/Stats.js

```diff
@@ -1,7 +1,27 @@
 import React from "react";
+import { Row, Col, Container } from "react-bootstrap";
 
 const Stats = () => {
-	return <div>Stats</div>;
+	return (
+		<div>
+			<Container>
+				<Row className='font-lexend text-4xl decoration-black font-semibold'>
+					<Col xs={4} md={3} className='flex justify-center'>
```

Comment:
Best way to solve the responsiveness would be to have the following sizes `sm={12} md={6} lg={3}` Doesnt seem to break at those numbers